test/infrastructure/docker/e2e/docker_suite_test.go

```diff
@@ -72,7 +78,11 @@ var _ = BeforeSuite(func() {
 	Expect(infrav1.AddToScheme(scheme)).To(Succeed())
 
 	// Create the management cluster
-	mgmt, err = NewClusterForCAPD(ctx, "mgmt", scheme, managerImage, capiImage)
+	kindClusterName := os.Getenv("MGMT_CLUSTER_NAME")
```

Comment:
```suggestion
	kindClusterName := os.Getenv("CAPI_MGMT_CLUSTER_NAME")
```

Either that or some other prefix